cloudevents/spec.md

```diff
@@ -626,3 +626,31 @@ The following example shows a CloudEvent serialized as JSON:
     "data" : "<much wow=\"xml\"/>"
 }
 ```
+
+### Example of a CloudEvent with encrypted data
+
+The following example shows a CloudEvent serialized as JSON with line-breaks
+for display purposes only:
+
+```
+{
+    "specversion" : "1.0",
+    "type" : "PAYMENT.AUTHORIZATION.CREATED",
+    "source" : "https://paymentprocessor.example.com/",
+    "subject" : "c7bbb040-d458-4d47-82a8-45413f9f2d33",
+    "id" : "a978702e-ef48-4032-ac18-a057e0104076",
+    "time" : "2024-05-30T17:31:00Z",
+    "datacontenttype" : "application/jose",
+    "data" : "eyJhbGciOiJSU0EtT0FFUCIsImVuYyI6IkEyNTZHQ00ifQ.
```

Comment:
No, because the contenttype defines what data is.
I think data_base64 is a short hand for contenttype base64 and a data field.
The contenttype in this example "application/jose" is correct.

Reply:
actually, `data_base64` doesn't mean that the content-type is base64, rather, it just means that the `data` serialization within this CE uses base64 rather than raw JSON. For example, it's possible to have content-type be `application/json` but then use `data_base64` to serialize your json even if the json is as simple as `{}` (e.g. `e30K`)